packages/flame/lib/src/game/render_context.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:ui';
+
+import 'package:meta/meta.dart';
+import 'package:vector_math/vector_math_64.dart';
+
+/// Helper class that assists the rendering pipeline in Flame.
+///
+/// This class keeps track of all canvas transformations applied thus far,
+/// allowing downstream components to query this information.
+@experimental
```

Comment:
I don't think we can consider this `experimental`, firstly because I personally think this is indeed the way forward for us but also because it will by default be exposed to users when they want to play with the `renderTree`, which is not an experimental method.